src/main/java/com/assure/movie/common/errors/ErrorAdvice.java

```diff
@@ -0,0 +1,31 @@
+package com.assure.movie.common.errors;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+/**
+ * @author Israel Yasis
+ */
+@ControllerAdvice
```

Comment:
You are using the @ControllerAdvice here but exists a @RestControllerAdvice too.
What is the difference between them? And which one do you have to you in this case?

Reply:
The @ControllerAdvice and @RestControllerAdvice are the same, but the @RestControllerAdvice include a @ResponseBody and this is why is more used in rest responses, so I should use @RestControllerAdvice.

Reply:
You are right, thanks!